src/ClassFinder.php

```diff
@@ -71,13 +71,10 @@ private static function getClassMap(string $namespace): array
             assert(!is_bool(self::$optimisedClassMap));
             return self::$optimisedClassMap ;
         }
-        $projectDirs = self::getProjectSearchDirs($namespace);
-        $map = [];
-        // Use composer's ClassMapGenerator to pull the class list out of each project search directory
-        foreach ($projectDirs as $dir) {
-            $map = array_merge($map, ClassMapGenerator::createMap($dir));
-        }
-        return $map;
+        return array_reduce(self::getProjectSearchDirs($namespace),
+            function($map, $dir) {
+                return array_merge($map, ClassMapGenerator::createMap($dir));
```

Comment:
Restore the removed comment to make it stone obvious to lesser minds than you what the blazes you're doing?

Reply:
Done